tests/PySys/mqtt_port/mqtt_port_unset/run.py

```diff
@@ -0,0 +1,55 @@
+import sys
+import time
+
+from pysys.basetest import BaseTest
+
+
+"""
+Validate changing the mqtt port using the tedge command
+
+Given a configured system
+When mqtt.port is set using tedge mqtt.port set with sudo
```

Comment:
```suggestion
When `mqtt.port` is set using `tedge mqtt.port set` with `sudo`
```

Could we make commands and other distinctive parameters marked some how, maybe similar as above?

Reply:
ok